### DISCUSSION TITLE
MAESTRO DETALLE

### DISCUSSION THREAD

**Discussion by arturoserafini** [Laravel-Backpack/community-forum#1243] [Q&A (Help)]
Se puede realizar maestro detalle con Backpack?

[a comment by an automated account is not shown]

**Comment by jcastroa87** [comment]
hola @arturoserafini 

Como nuestro Bot indica, sí, es posible generar un Maestro-Detalle en Backpack, principalmente mediante las [relaciones](https://backpackforlaravel.com/docs/6.x/crud-fields#relationship-pro), en Bacpack Pro existe este tipo de campo, que facilita mucho el desarrollo, también ofrece la opción de [InlineCreate](https://backpackforlaravel.com/docs/6.x/crud-fields#create-related-entries-in-a-modal-using-the-inlinecreate-operati).

Además puedes aprovechar esta semana de Black Week para aprovechar de comprar la versión Pro con [increíbles descuentos aquí](https://backpackforlaravel.com/articles/news/black-week-discounts-hurry-up?utm_source=digest&utm_medium=email).

Saludos.

**Comment by arturoserafini** [comment]
Muchas gracias... estaré leyendo la documentación..
